docs/vdc/networking-virtual-datacenter.md

```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 
 #### Connectivity to the cloud
-The VDC needs connectivity with external networks to offer services to customers, partners and/or internal users. This usually means connectivity not only to the Internet, but also to on-premises networks and data centers.
+The VDC needs connectivity with external networks to offer services to customers, partners, and internal users. It usually needs connectivity not only to the internet but also to on-premises networks and datacenters.
 
-Customers can build their security policies to control what and how specific VDC hosted services are accessible to/from the Internet using [Azure Firewall][AzFW] or Network Virtual Appliances, and custom routing policies and network filtering ([User Defined Routing][UDR] and [Network Security Groups][NSG]). It's recommended that all Internet facing resources be additionally protected by [**Azure DDOS Protection Standard**][DDOS].
+Customers can build their security policies to control what and how specific VDC-hosted services are accessible to or from the internet with [Azure Firewall][AzFW] or network virtual appliances, custom routing policies, and network filtering. See information on [user-defined routes][UDR] and [network security groups][NSG]. We recommend that all internet-facing resources be additionally protected by [Azure DDoS Protection Standard][DDOS].
 
-Enterprises often need to connect VDCs to on-premises data centers or other resources. The connectivity between Azure and on-premises networks is therefore a crucial aspect when designing an effective architecture. Enterprises have two different ways to create an interconnection between VDC and on-premises in Azure: transit over the Internet and/or by private direct connections.
+Enterprises often need to connect VDCs to on-premises datacenters or other resources. So the connectivity between Azure and on-premises networks is a crucial aspect when designing an effective architecture. Enterprises have two different ways to create interconnections between VDCs and on-premises in Azure: transit over the internet or private direct connections.
 
-An [**Azure Site-to-Site VPN**][VPN] is an interconnection service over the Internet between on-premises networks and the VDC, established through secure encrypted connections (IPsec/IKE tunnels). Azure Site-to-Site connection is flexible, quick to create, and does not require any further procurement, as all connections connect over the internet.
+An [Azure site-to-site VPN][VPN] is an interconnection service over the internet between on-premises networks and the VDC. It's established through secure encrypted connections, IPsec/IKE tunnels. An Azure site-to-site connection is flexible and quick to create. This VPN doesn't require any further procurement as all connections connect over the internet.
```

Comment:
Should be "An [Azure site-to-site VPN][VPN] connects an Enterprise's on-premises network to their VDC implementation over the public internet. Traffic over the VPN is encrypted using IPSec and IKE tunneling."